Plugins/Util/NWScript/nwnx_util.nss

```diff
@@ -163,6 +163,15 @@ int NWNX_Util_RegisterServerConsoleCommand(string sCommand, string sScriptChunk)
 /// @param sCommand The name of the command.
 void NWNX_Util_UnregisterServerConsoleCommand(string sCommand);
 
+/// @brief Determines if the given plugin exists and is enabled.
+/// @param sPlugin The name of the plugin to check.
```

Comment:
Should probably note that it expects the plugin name as `NWNX_{Name}`, case-sensitive and all.

Unless you want to go the extra mile and strip the prefix only when needed and use `GetServices()->m_plugins->GetCanonicalPluginName()` to deal with case-sensitivity. :P

Reply:
I will add a comment ;)

Removing the ` NWNX_*` prefix might be a bit much, I think it makes sense to match the plugin naming used in `NWNX_Core`.